### PR TITLE
Updating operator installation instruction

### PR DESCRIPTION
**Pull request by dewan-ahmed**
Updating operator installation instruction for v1.0.1 --> v1.1.1 upgrade path.

### CODE REVIEW

**Review by wtam2018 — APPROVED** [review]
LGTM
thanks @dewan-ahmed

File: modules/installing-gitops-operator-in-web-console.adoc [hunk 1]
```diff
@@ -18,3 +18,7 @@
 Red Hat OpenShift GitOps will be installed in all namespaces of the cluster.
 
 Once the Red Hat OpenShift GitOps Operator is installed, it automatically sets up a ready-to-use Argo CD instance that is available in the `openshift-gitops` namespace, and an Argo CD icon is displayed in the console toolbar.
+
+If you have OpenShift GitOps Operator already installed on your cluster, the existing installation is seamlessly upgraded. 
+
+NOTE: If you're upgrading from v1.0.1, please use the **stable** channel which enables installation of the latest stable and supported release of the Red Hat OpenShift GitOps Operator.
```
Comment (Preeticp) [inline modules/installing-gitops-operator-in-web-console.adoc:24]: We use active voice where possible, please change this to: 'If you upgrade'
It also removes the you're contraction which we do not use in the docs. 
Remove the word 'please', we need to be authoritative - as in - you must do x, y, z
Either replace 'which' with 'that' or add a comma after channel. When you use which, it needs to be preceded with a comma.

**Comment by Preeticp** [comment]
Hi @dewan-ahmed thank you for the PR. I have left a few minor suggestions, can you fix them, please?
Also do we plan to add instructions to the installation similar to how we have for Pipelines here: https://github.com/dewan-ahmed/openshift-docs/blob/enterprise-4.7/modules/op-installing-pipelines-operator-in-web-console.adoc

**Comment by dewan-ahmed** [comment]
> Hi @dewan-ahmed thank you for the PR. I have left a few minor suggestions, can you fix them, please?
> Also do we plan to add instructions to the installation similar to how we have for Pipelines here: https://github.com/dewan-ahmed/openshift-docs/blob/enterprise-4.7/modules/op-installing-pipelines-operator-in-web-console.adoc

Hi @Preeticp I pushed another commit after the minor fixes.

Re: following pipelines docs model, I have asked @wtam2018 about his thoughts.

**Pull request closed without merging**